index.html

```diff
@@ -0,0 +1,139 @@
+         </ul>
+         <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+      </nav>
+  </header>
+                         <!--First Section-->
+  <section class="headlineSection">
+    <div class="content">
+    <p class="transparentText">"Let The Music Speak !"</p>
+    <h1 class="title">The music festival<br>Hangang Summer <br> 2022 </h1>
+    <div class="description">
+     <p>2022 marks the seventh year of the Hangang Summer Festival and we have planned a wide selection of over 80 programs that are designed to allow you to be a part of this grand picnic while enjoying 
+       the beautiful natural surroundings of Seoul’s landmark Hangang River.</p> 
+    </div>
+    <p class="date">2022 juin 25 (Sat)-11 july (Mon)</p>
+    <p class="place">@Bop Club and Bar and More</p>
+  </div>
+  </section>
+
+                            <!--Program section-->
+  <section class="program">
+  <h2 class="titre">Main program</h2>
+  <hr>
+  <div class="cards">
+    <div class='card'>  
+      <div class="prgimg">
+        <i class="fa-solid fa-microphone fa-2xl prgimg"></i>
+      </div>
+      <h3>Open entry</h3>
+      <p class="text">Authoritatively initiate worldwide services through premier best practices.</p>
+    </div>
+    <div class='card'>
+      <div class="prgimg">
+        <i class="fa-solid fa-champagne-glasses fa-2xl prgimg"></i>
+      </div>
+      <h3>Boom Party</h3>
+      <p class="text">Authoritatively initiate worldwide services through premier best practices.</p>
+    </div>
+    <div class='card'>
+      <div class="prgimg">
+        <i class="fa-solid fa-burger fa-2xl prgimg"></i>
+     </div>
+      <h3>Buffet Dinner</h3>
+      <p class="text">Authoritatively initiate worldwide services through premier best practices.</p>
+    </div>
+    <div class='card'>
+      <div class="prgimg">
+        <i class="fa-solid fa-compact-disc fa-2xl prgimg"></i>
+      </div>
+      <h3>pink party</h3>
+      <p class="text">Authoritatively initiate worldwide services through premier best practices.</p>
+    </div>
+    <div class='card'>
+      <div class="prgimg">
+        <i class="fa-solid fa-guitar fa-2xl prgimg"></i>
+    </div>
+      <h3>Black Band</h3>
+      <p class="text">Authoritatively initiate worldwide services through premier best practices.</p>
+    </div>
+  </div>
+    <button class="programButton">Reserve Seat Now</button>
+  </section>
+  
+                         <!--Dj section-->
+  <section class="dj">
+       <h2  class="titre">Featured Djs</h2>
+       <hr>
+       <div class='djitems'>
+       </div>
+  </section> 
+                            <!--Sponsor section-->
+    <section class="Sponsor">
+      <h3>Partners</h3>
+      <hr>
+    <ul class="sponsorList">
+      <li class="design sponsorImg"><img src="images/cola.jpg" alt="cola"></li>
```

Comment:
- [ ] Please fix these issues for the partner's section and make sure it matches the design on the template
  - The images are a little bigger in the partner's section, Please decrease the size of the images.
  - Please apply `grayscale` properties. (For grayscale effect you can use this code, `-webkit-filter: grayscale(100%); moz-filter: grayscale(100%); filter: grayscale(100%); filter: gray;` )
  - Please use the correct background color. `#272a31`

### Your design

![image](https://user-images.githubusercontent.com/83803414/163356779-d54e2bd7-f832-49b4-b2c3-cfce064e510c.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/163356815-da5b84dc-013d-465a-bc28-7c9e50d2fec9.png)